Dynatrace/dynatrace-operator
# Improve kuttl test suit

# Description

Simple improvements for our kuttl tests: 
- Add makefile commands for kuttl
- Add check for required env variables in makefile
- Correctly set timeout in our teststeps
- Updated kind node images

## How can this be tested?

Run 
- `make create-kind-cluster` - Kind cluster should be created
- `make kuttl-oneagent` - All tests should pass

## Checklist
